### DISCUSSION TITLE
clone

### DISCUSSION THREAD

**Discussion by amineamdouni** [community/community#66770] [Repositories] [answered]
### Select Topic Area

Question

### Body

how to create fork

**Comment by loneicewolf** [comment, marked as answer]
## 1. Go to the repo you want to clone/fork
## 2. click the Fork button :)
## 3. for example, on my readme page:
![image](https://github.com/community/community/assets/68499986/6bed433e-393b-4a04-bb4b-148b48b425df)

**Reply by loneicewolf** [reply]
# Have a great day! :) Wishes from Sweden

**Reply by loneicewolf** [reply]
oh and if it's grayed out  like in the picture, it is because simply one can't fork this specific repo. This is only the case in rare scenarios. `:-)`
